src/app/components/tables/group-function/terra-group-function.component.ts

```diff
@@ -62,6 +64,15 @@ export class TerraGroupFunctionComponent
     @Language()
     protected lang:string;
 
+    public ngOnInit():void
+    {
+        // implementation is required by angular-l10n. See https://robisim74.github.io/angular-l10n/spec/getting-the-translation/#messages
+    }
+
+    public ngOnDestroy():void
+    {
+        // empty and needed for l10n
```

Comment:
Die Texte sollten in `ngOnInit()` und `ngOnDestroy()` identisch sein. 

Der aus `ngOnInti()` bietet mehr infromationen und sollte deswegen bevorzugt werden 😄